pkgs/applications/networking/mailreaders/neomutt/default.nix

```diff
@@ -73,42 +73,46 @@ stdenv.mkDerivation rec {
   ''
   # https://github.com/neomutt/neomutt-contrib
   # Contains vim-keys, keybindings presets and more.
-  + lib.optionalString withContrib "${lib.getExe lndir} ${passthru.contrib} $out/share/doc/neomutt";
+  + lib.optionalString withContrib "${lib.getExe lndir} ${finalAttrs.passthru.contrib} $out/share/doc/neomutt";
 
-  doCheck = true;
+  doCheck = false;
```

Comment:
I don't think you intend to disable `doCheck` and fix `preCheck` in the same PR, do you ?
If this is what you really want, please document why you want to disable `doCheck`

Reply:
Thanks for calling this out, @nim65s, I'l have a l👀k

Reply:
Took me a while to get to this, @nim65s, yet I believe I've found an acceptable approach and kindly re-request review.